src/components/search-bar/story-header.dev.inc.tsx

```diff
@@ -24,6 +27,26 @@ const StoryHeader: React.FC<StoryHeaderProps> = ({ search, userProfile }) => {
   const [isEnabledAdvancedSearch, setIsEnabledAdvancedSearch] = useState(true);
   const headerMenuRef = React.useRef<HTMLDivElement>(null);
 
+  const googleTranslateElementInit = () => {
```

Comment:
Move the init function definition from the component, it's does not use the component context.